README.md

```diff
@@ -80,9 +82,12 @@ The library leverages the following environment variables:
 
 - Store any Jinja2 variable definitions you wish to use in your configuration
   template files in `resources/myapp.yml`.
```

Comment:
let's rename `myapp.yml` here and in the above to `variables.yml`

Reply:
`settings.yml`